docs/stencil-docs/storefront-customization/using-custom-fonts-and-icons.md

```diff
@@ -177,13 +178,12 @@ Then, from your theme's root directory, run:
 
 `npm install`
 
-### Calling Theme Icons
+### Calling theme icons
 
 A Stencil theme's icons are delivered via a single SVG sprite, which is embedded on the page in
 `<theme-name>templates/layout/base.html`. This sprite is generated via the Grunt task `grunt svgstore`.
 
-The task takes individual SVG files for each icon (in the theme's `assets/icons/` subdirectory) and bundles
-them together, to be inlined on the top of the theme, inside a Handlebars partial.
+The task takes individual SVG files for each icon (in the theme's `assets/icons/` subdirectory) and bundles them together, to be inlined on the top of the theme, inside a Handlebars partial.
 
 You can then call each icon in a similar way to an inline image, via the following HTML:

```

Comment:
You can then call each icon in a similar way to an inline image via the following HTML: